pretyping/evarconv.ml

```diff
@@ -1653,6 +1665,11 @@ let is_beyond_capabilities = function
   | CannotSolveConstraint (pb,ProblemBeyondCapabilities) -> true
   | _ -> false
 
+let is_constant_instance sigma (evk, args) alias =
+  let args = Evd.expand_existential sigma (evk, args) in (* FIXME *)
```

Comment:
Either fix or make the comment more explicit about what needs fixing

Reply:
All the FIXMEs are about removing expand_existential. I'll just remove the comments.